docs/admin/system-information.rst

```diff
@@ -829,42 +829,67 @@ flag::
 Description of checked node settings
 ------------------------------------
 
+.. raw:: html
+
+  <span id="recovery-expected-nodes"></span>
+  
 Recovery expected data nodes
 ............................
 
-The check for the
-:ref:`gateway.expected_data_nodes <gateway.expected_data_nodes>` setting checks
-that the number of data nodes that should be waited for the immediate
-cluster state :ref:`recovery <gloss-shard-recovery>`. This value must be equal
-to the maximum number of data nodes in the cluster.
+This check looks at the 
+:ref:`gateway.expected_data_nodes <gateway.expected_data_nodes>` setting and 
+checks if the setting matches the actual number of data nodes present in the 
+cluster. 
+The value of this setting is compared against 
+:ref:`gateway.recover_after_data_nodes
+<gateway.recover_after_data_nodes>` (or the deprecated 
+:ref:`gateway.recover_after_nodes <gateway.recover_after_nodes>`)
+during startup to decide if the cluster can start with 
+:ref:`recovery <gloss-shard-recovery>`.
+If the actual number of nodes is below the expected number of nodes, 
```

Comment:
I think that the usage of the word `flagged` is confusing, I cannot understand how exactly the 2 settings are related and when the recover can start or not.

Reply:
Thank you for flagging this @matriv , that was indeed confusing as it was mixing things from the health check that is explained in the section that comes just after.
I have [hopefully] fixed this in https://github.com/crate/crate/pull/13567/commits/93b75ba5f61a28e482be8f2e14713f48ac4beb6a